appservice/src/createAppService/SiteStep.ts

```diff
@@ -199,6 +199,34 @@ export class SiteStep extends WizardStep {
             {
                 name: 'node|8.1',
                 displayName: 'Node.js 8.1'
+            },
+            {
+                name: 'php|5.6',
+                displayName: 'PHP 5.6'
+            },
+            {
+                name: 'php|7.0',
+                displayName: 'PHP 7.0'
+            },
+            {
+                name: 'php|7.2',
+                displayName: 'PHP 7.2'
```

Comment:
I don't see 7.2 listed?
<img width="531" alt="screen shot 2018-02-27 at 4 57 48 pm" src="https://user-images.githubusercontent.com/11282622/36764014-731cc728-1bdf-11e8-8e54-e37b747f3446.png">

Reply:
Portal app creation and Portal application settings don't match.  It seems that PHP 7.2 is supported though, so +1 for extension creation vs. portal creation.

![image](https://user-images.githubusercontent.com/5290572/36806877-4f7b122e-1c76-11e8-98bf-ca88ef8cff7b.png)

Reply:
@nturinski if all of those node versions are supported - can you add them as well? And make 8.9 the LTS?